src/maven-repair/pom.xml

```diff
@@ -273,7 +279,7 @@
                 <dependency>
                     <groupId>fr.inria.gforge.spirals</groupId>
                     <artifactId>nopol</artifactId>
-                    <version>666abb764bf1819f6c316faf4fe5b559ac583de1</version>
+                    <version>0f47cdb2556b8003d717c62325fef4dbf09a2bae</version>
```

Comment:
could you add a comment about the release date of this one?

Reply:
Done!